the-list-ui/src/App.js

```diff
@@ -1,7 +1,11 @@
 import logo from './logo.svg';
 import './App.css';
+import { getStudents } from './request-functions.js';
 
 function App() {
+  const backEndURL = 'http://localhost:4000';
```

Comment:
```suggestion
  const backEndURL = process.env.REACT_APP_BACKEND_URL;
```

Чтобы это работало, надо создать в `the-list-ui` файлик `.env` и поместить в него:
```
REACT_APP_BACKEND_URL = http://localhost:4000
```

Это позволит нам поменять URL бакенда для продакшена отредактировав `.env`-файл и не внося правки в код.

Вы можете также иметь `.env.local` файл со своими локальными настройками. `.env.local` переопределяет `.env`. `.env.local` коммитить не надо.

Reply:
Также, можно эту константу сделать глобальной, вынести за пределы функции `App`.

Reply:
.env файл надо создать в папке `the-list-ui`?